signing-providers/package.json

```diff
@@ -10,15 +10,15 @@
   "dependencies": {
     "@multiversx/sdk-core": "13.7.0",
     "@multiversx/sdk-dapp-utils": "0.1.0",
-    "@multiversx/sdk-extension-provider": "3.0.0",
+    "@multiversx/sdk-extension-provider": "4.0.0",
     "@multiversx/sdk-hw-provider": "7.0.0",
     "@multiversx/sdk-native-auth-client": "1.0.9",
     "@multiversx/sdk-wallet": "4.5.1",
     "@multiversx/sdk-wallet-connect-provider": "5.0.0",
     "@multiversx/sdk-web-wallet-cross-window-provider": "2.0.1",
     "@multiversx/sdk-web-wallet-iframe-provider": "2.0.0",
-    "@multiversx/sdk-web-wallet-provider": "3.2.1",
-    "@multiversx/sdk-webview-provider": "1.0.0",
+    "@multiversx/sdk-web-wallet-provider": "4.0.0",
```

Comment:
I am not sure what's happening with the delta. It should have been v4.1.0.

https://github.com/multiversx/mx-sdk-js-examples/pull/49

Reply:
Conflicts have to be solved first.